godtools/App/AppDiContainer.swift

```diff
@@ -176,8 +176,7 @@ class AppDiContainer {
             snowplowAnalytics: SnowplowAnalytics(config: config, userAuthentication: userAuthentication, loggingEnabled: analyticsLoggingEnabled)
         )
                                                           
-        openTutorialCalloutCache = OpenTutorialCalloutUserDefaultsCache()
-                           
+        openTutorialCalloutCache = OpenTutorialCalloutUserDefaultsCache(getTutorialIsAvailableUseCase: GetTutorialIsAvailableUseCase(deviceLanguage: deviceLanguage))
```

Comment:
Hey @rachaelblue for this we don't want a useCase (domain layer) getting injected into a data layer class.  It's probably not very clear to you with the services folder containing mixed data and domain logic.  That's something I would like to eventually make more distinct in GodTools as we refactor those services. 
What's your schedule looking like today? It would be good to quickly connect on this one.